src/main/resources/static/_shared.yml

```diff
@@ -3669,7 +3669,6 @@ components:
         - id
         - level
         - fileName
-        - createdAt
```

Comment:
This can be null from `AP and Delius API` so had no choice but to make it nullable int the open-api spec